docs/alternate_domains.rst

```diff
@@ -52,6 +52,8 @@ This service is generously provided by Cloudflare.
     or another variation. While these continue to resolve,
     they do not yet allow us to acquire SSL certificates for those domains.
     Simply point the CNAME to ``readthedocs.io``.
+    
+    If you change the CNAME, the SSL certificate issuance will take about one hour.
```

Comment:
nitpick: I prefer "can take" rather than "will take" especially since for a new CNAME (as opposed to a changed one) this should be almost immediate.